cmd/operator/main.go

```diff
@@ -303,6 +303,15 @@ func run() int {
 		return 1
 	}
 
+	kubernetesVersion, err := kclient.Discovery().ServerVersion()
```

Comment:
Instead of repeating the same check in each controller, let's do it once on startup.